app/controllers/reliefs/AvoidanceSchemeBeingUsedController.scala

```diff
@@ -68,20 +69,21 @@ trait AvoidanceSchemeBeingUsedController extends BackLinkController
           } yield result
         }
       }
+    }
   }
 
-  def send(periodKey: Int) = AuthAction(AtedRegime) {
-    implicit atedContext =>
+  def send(periodKey: Int): Action[AnyContent] = Action.async { implicit request =>
+    authorisedAction { implicit authContext =>
       ensureClientContext {
         validatePeriodKey(periodKey) {
-          val data = AtedUtils.addParamsToRequest(atedContext, Map("periodKey" -> ArrayBuffer(periodKey.toString)))
+          val data = AtedUtils.addParamsToRequest(Map("periodKey" -> ArrayBuffer(periodKey.toString)))
           isTaxAvoidanceForm.bindFromRequest(data.get).fold(
             formWithError =>
               currentBackLink.map(backLink =>
                 BadRequest(views.html.reliefs.avoidanceSchemeBeingUsed(periodKey, formWithError, PeriodUtils.periodStartDate(periodKey), backLink))
               ),
             isTaxAvoidance => {
-              reliefsService.saveDraftIsTaxAvoidance(atedContext.user.atedReferenceNumber, periodKey, isTaxAvoidance.isAvoidanceScheme.getOrElse(false)).
+              reliefsService.saveDraftIsTaxAvoidance(authContext.atedReferenceNumber.get, periodKey, isTaxAvoidance.isAvoidanceScheme.getOrElse(false)).
```

Comment:
RED - None.get risk.

Reply:
Removed and refactored to use String as opposed to Option String